src/arguments/CoreCategoryChannel.ts

```diff
@@ -0,0 +1,20 @@
+import { isCategoryChannel } from '@sapphire/discord.js-utilities';
+import type { PieceContext } from '@sapphire/pieces';
+import type { GuildChannel, CategoryChannel } from 'discord.js';
```

Comment:
Peut-être que je pinaille un peu but could you order these alphabetically so that it's `import type { CategoryChannel, GuildChannel } from 'discord.js';`?